src/test/java/io/stargate/sgv2/jsonapi/testresource/StargateTestResource.java

```diff
@@ -73,6 +73,14 @@ public Map<String, String> start() {
       propsBuilder.put("stargate.int-test.coordinator.cql-port", cqlPort);
       propsBuilder.put("stargate.int-test.cluster.persistence", getPersistenceModule());
 
+      // Many ITs create more Collections than default Max 5, use more than 50 indexes so:
```

Comment:
Added this after observing majority of ITs failing with 5 Collection max: but with 10 Collections all pass.
May need to be tuned over time if ITs create more Collections.